models/articles.models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.readArticles = (article_id) => {
+    if( Number.isNaN(Number(article_id))){
+        return Promise.reject({ status: 400, msg:'Bad Request: article_id is not a number' })
+    }
```

Comment:
You don't need to check whether the article_id is a number or not, PSQL will return an error if you try to query an article_id with a different data type. Then you can catch the error in the controller, send it to next and by using the error code create a error handler that will send this same response to the client. 

By setting this up with error handlers etc, you will be able to use it again and again for other endpoints.